src/lib/components/Menu/StyledMenu.tsx

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React from 'react';
+import { Menu, MenuItem, MenuProps, withStyles } from '@material-ui/core';
+
+export const StyledMenu = withStyles((theme) => ({
```

Comment:
do you get correct props types when using StyledMenu elsewhere?

Reply:
Yeah, StyledMenu is just a wrapper of Menu, but with styles. The interface doesn't change at all